piksi_tools/console/settings_view.py

```diff
@@ -212,6 +227,40 @@ def _factory_default_button_fired(self):
                         + "Are you sure you want to reset to factory defaults?"
     confirm_prompt.run(block=False)
 
+  def _settings_save_to_file_fired(self):
```

Comment:
I think we should change the logic. When a setting is changed in the UI it should be saved to flash also at that point. So we do not have any differences between UI/Flash settings. It was a bit confusing for me at the start what happens and I think it is the same for a random user who does not know the code.